tests/lib/cisco_test.py

```diff
@@ -640,6 +640,30 @@ def testDsmo(self):
 
     self.naming.GetNetAddr.assert_called_once_with('SOME_HOST')
 
+  def testDsmo2(self):
```

Comment:
You actually want this in the summarizer_test.py file since you are testing the summarizer, I realize there are some dsmo tests in cisco_test.py but this really should live there instead.